### PR TITLE
update GNOME for 15 SP6

### PR DESCRIPTION
**Pull request by jfaltenbacher**
PR creator: Description

Update GNOME user guide for 15 SP6 to GNOME version 45.3
Changed descriptions and screenshots were needed.
Run optipng
Run spellcheck
Run stylecheck
Run linkcheck

PR creator: Are there any relevant issues/feature requests?

* https://jira.suse.com/browse/DOCTEAM-1354

PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP5/openSUSE Leap 15.5
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### CODE REVIEW

File: xml/gnome_start.xml [hunk 1]
```diff
@@ -540,21 +484,16 @@
...
+    The <guimenu>Log Out</guimenu> and <guimenu>Switch User</guimenu>
     entries only appear in the menu if you have more than one user account on
     your system.
     </para>
-   
-   <procedure>   
+   <procedure>
     <step>
      <para>
       Click the <guimenu>System</guimenu> menu in the upper right corner of the
```
Comment (dariavladykina) [inline xml/gnome_start.xml:494]:
```suggestion
      Click the <guimenu>System</guimenu> menu in the upper-right corner of the
```

File: xml/gnome_start.xml [hunk 2]
```diff
@@ -591,12 +530,8 @@
...
```
Comment (dariavladykina) [inline xml/gnome_start.xml:532]:
```suggestion
      Click the <guimenu>System</guimenu> menu in the upper-right corner of the
```

File: xml/gnome_start.xml [hunk 3]
```diff
@@ -518,6 +449,19 @@
...
+   <listitem>
+    <formalpara>
+      <title>
+        <guimenu>Log out</guimenu> or <guilabel>Switch user</guilabel>
+      </title>
+      <para>
+        Finish the current session, but leave the computer on, so other users
+        can log in.
+        This option is only available if you have more than one user account on
+        your
```
Comment (dariavladykina) [inline xml/gnome_start.xml:461]:
```suggestion
        your system.
```

File: xml/gnome_start.xml [hunk 4]
```diff
@@ -518,6 +449,19 @@
      </para>
     </formalpara>
    </listitem>
+   <listitem>
+    <formalpara>
+      <title>
+        <guimenu>Log out</guimenu> or <guilabel>Switch user</guilabel>
```
Comment (dariavladykina) [inline xml/gnome_start.xml:455]:
```suggestion
        <guimenu>Log Out</guimenu> or <guilabel>Switch User</guilabel>
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions and a comment here. Thanks!

File: xml/gnome_accessibility.xml [hunk 5]
```diff
@@ -30,9 +30,10 @@
 
   <para>
    To configure accessibility features, open the <guimenu>Settings</guimenu>
-   dialog by right-clicking the desktop and choosing <guimenu>Settings</guimenu>.
-   In the navigation pane, select <guimenu>Accessibility</guimenu>. Each
-   assistive feature can be enabled separately.
+   dialog box by right-clicking the desktop and choosing
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply xml/gnome_accessibility.xml:33]: it was proposed by the vale style checker - so I thought I am on the safe side :/ I changed it now in TermWeb and will notify Tomas about this so he can change for vale

[the remark this answers is not in the document]
Comment (dariavladykina) [reply xml/gnome_accessibility.xml:33]: Very well! Thanks a lot!

File: xml/gnome_crypto.xml [hunk 6]
```diff
@@ -715,8 +715,8 @@
    </step>
    <step>
     <para>
-     Decide if you want to revoke your signature at a later date and if you want
-     to make your signature public.
+     Decide to revoke your signature at a later date and to make your signature
+     public.
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply xml/gnome_crypto.xml:719]: removing 'if you want' was proposed by the stylechecker. I reverted the change

File: xml/gnome_start.xml [hunk 7]
```diff
@@ -233,13 +224,12 @@
    </para>
    <para>
     The <guimenu>Activities</guimenu> overview allows you to access your
-    windows, applications, files, folders, and the web.
+    windows, applications, files, folders and the web.
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply xml/gnome_start.xml:227]: changed as suggested

File: xml/gnome_start.xml [hunk 8]
```diff
@@ -459,34 +412,12 @@
  </sect1>
  <sect1 xml:id="sec-gnomeuser-start-logout">
   <title>Pausing or finishing your session</title>
-
   <para>
    To pause or close your session, open the <guimenu>System</guimenu> menu
-   in the upper right corner of the top bar and choose one of the options
-   listed.
+   in the upper right corner of the top bar, click the
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply xml/gnome_start.xml:417]: i belive it was also proposed by the stlychecker to remove the hyphen. Reverted these changes

File: xml/gnome_start.xml (hunk 3 above)
Comment (jfaltenbacher) [reply xml/gnome_start.xml:461]: changed as suggested - good catch

File: xml/gnome_crypto.xml (hunk 6 above)
[the remark this answers is not in the document]
Comment (dariavladykina) [reply xml/gnome_crypto.xml:719]: Yep, it has some debatable tricks :-)

File: xml/gnome_start.xml (hunk 4 above)
Comment (jfaltenbacher) [reply xml/gnome_start.xml:455]: changed as suggested

**Comment by jfaltenbacher** [comment]
update GNOME for 15 SP 6

**Pull request merged**